AirCasting/Graph/GraphView.swift

```diff
@@ -40,6 +44,17 @@ struct GraphView: View {
 
 struct GraphView_Previews: PreviewProvider {
     static var previews: some View {
-        GraphView(thresholds: .constant([0,1,2,3]))
+        GraphView(thresholds: [.mock])
     }
 }
+
+struct ObservedObjectWrapper<T: ObservableObject, Content: View>: View {
```

Comment:
Is this related to Graf? Seams like `Lazy` is the word that would describe this better

Reply:
I think I don't understand your comment here.
What I use here is a convenient view wrapper that lets me observe changes to a single object from thresholds array.
It's impossible to mark array as an `@ObservedObject`, and I have to observe changes there so I needed a view for each of `thresholds` elements.
I should have added a comment there. I'm adding one now. :)

Reply:
After our call I removed it to see why I implemented it in the first place...and everything is working just fine. It seems like there was a problem somewhere else.